src/modules/Auth/features/Login/LoginForm/LoginForm.tsx

```diff
@@ -74,9 +100,28 @@ export const LoginForm = () => {
         replace: true,
       });
     }
+
+    // Display soft-lock state
+    if (softLockData) {
+      dispatch(banners.actions.addBanner({ key: 'SoftLockWarningBanner' }));
+      setFocus('password');
+
+      // Refocus password field when window regains focus
+      const handleWindowFocus = () => {
+        setFocus('password');
+        window.removeEventListener('focus', handleWindowFocus);
+      };
+
+      window.addEventListener('focus', handleWindowFocus);
+
+      return () => {
+        window.removeEventListener('focus', handleWindowFocus);
+      };
+    }
   }, []);
```

Comment:
I know you only modified this hook, and did omit the dependencies yourself, but so long as we're editing it, and so long as we know that we only want this hook to run on mounting/unmounting of the `LoginForm` — perhaps we can disable the lint warning here? 

It would be great to take opportunities to chip away at the mountain of them when the opportunity arises 😅.